configuration/configs/development.json

```diff
@@ -414,10 +414,10 @@
   "agent": {
     "rinkeby": {
       "rpcStyle": "ethereum",
-      "db": "db",
+      "db": "/usr/share/nomad",
```

Comment:
NOTE: Since these values cannot be overridden with env variables at this moment, we had to update this value with what our containers mounts the volume to.